src/main.ts

```diff
@@ -2,13 +2,25 @@ import 'source-map-support/register';
 import { logger } from './modules/common/Logger';
 import Stoa from './Stoa';
 
+import { ArgumentParser } from "argparse";
 import express from 'express';
 import { URL } from 'url';
 
-const port: number = Number(process.env.PORT) || 3836;
-const agora_address: URL = new URL(process.env.AGORA_ENDPOINT || "http://127.0.0.1:2826");
-// TODO: Replace hardcoded "127.0.0.1" with  agora_address.hostname (#106)
-const stoa: express.Application = new Stoa("database", "127.0.0.1", agora_address.port).stoa;
+// Parse the arguments
+const parser = new ArgumentParser();
+parser.add_argument('-a', '--agora', { help: "The endpoint of Agora" });
+parser.add_argument('-p', '--port', { help: "The network port of Stoa" });
+parser.add_argument('-d', '--database', { help: "The file name of sqlite3 database" });
+let args = parser.parse_args();
+
+const port: number = Number(args.port || "3836");
+const agora_address: URL = new URL(args.agora || "http://127.0.0.1:2826");
```

Comment:
If the address is only in agora_address, port exception processing will be required.
For example, if `-a http://127.0.0.1`
In this case, we should apply the default port value.

Reply:
If the port is not set, it is being treated as the http default of 80.

Reply:
Agora's connection information will be resolved by proceeding with Issue #85 to use endpoints without separating the host from the port.